src/libertem/web/server.py

```diff
@@ -230,17 +239,38 @@ def run(host, port, browser, local_directory, numeric_level, token, preload, str
 
     main(bound_sockets, event_registry, shared_state, token)
 
+    async def create_and_set_executor():
+        if executor_spec is not None:
+            # Must happen after main() so that Dask uses our event loop
```

Comment:
Comment might need an update - also an explanation that yes, we are blocking the event loop here, which we also might want to **try** to fix...

Reply:
I've updated the comment. LGTM ?